speaker_verification/sql_utils.py

```diff
@@ -52,12 +63,11 @@ def read_sqlite_table(table):
         cur.execute(sqlite_select_query)
         records = cur.fetchall()
         for row in records:
-            print("Id: ", row[0])
-            print("mfcc: ", type(row[1]))
-            print("\n")
+            logger.info("Id: ", row[0])
+            logger.info("mfcc: ", type(row[1]))
 
     except sqlite3.Error as error:
-        print("Failed to read data from sqlite table", error)
+        logger.info("Failed to read data from sqlite table", error)
```

Comment:
I think this should be logged as at least the WARNING level but to be consistant with the other logs I'd recommend just using ERROR.
```suggestion
        logger.error("Failed to read data from sqlite table", error)
```

Reply:
Yep you're right!